qiime/sdk/tests/test_visualization.py

```diff
@@ -369,6 +369,27 @@ def test_get_index_paths_multiple_html_from_data_dir(self):
         with self.assertRaises(ValueError):
             visualization.get_index_paths()
 
+    def test_get_index_path_relative_false(self):
+        data_dir = os.path.join(self.test_dir.name, 'multi-html-viz2')
+        os.mkdir(data_dir)
+        multi_html_viz(data_dir, [1, 42])
+
+        visualization = Visualization._from_data_dir(data_dir,
```

Comment:
this is shadowed below, I think you can remove this line and all previous lines in the method